runtime/vm/vmprops.c

```diff
@@ -1720,13 +1720,7 @@ getPlatformPropertyList(JNIEnv *env, const char *strings[], int propIndex)
 		convertToUTF8(PORTLIB, unicodeTemp, userdir, EsMaxPath);
 		strings[propIndex++] = userdir;
 	}
-#endif /* JAVA_SPEC_VERSION < 17 */
-
-	if (JAVA_SPEC_VERSION < 12) {
-		/* Get the timezone */
-		strings[propIndex++] = "user.timezone";
-		strings[propIndex++] = "";
```

Comment:
Did you mean to remove `user.timezone`?

Reply:
Nevermind, I see you moved it to `getSystemPropertyList()`.
The two definitions of `getPlatformPropertyList()` should be `static`.

Reply:
I moved it to the common code, we didn't need two copies for Windows and non-Windows.

Reply:
Made them static.